discord/momobot.py

```diff
@@ -71,6 +72,21 @@ def getFilenameFromUrl(url):
     return url[url.rindex('/')+1:]
 
 
+def toraList():
```

Comment:
Optional, but this is a great caching opportunity because we don't really need to make a network call to khoi's server for a list of filenames that will update only every now and then. We could store a successfully fetched toraList for maybe a day at a time without re-fetching it.

Reply:
Double extra bonus, instead of toraList() do something like fetchFromKhoiServer(prefix) where prefix can be tora or anything else

Reply:
See 42